### PR TITLE
Add fastlane to the project

### PR DESCRIPTION
**Pull request by robertmogos**

### CODE REVIEW

File: fastlane/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+installer/
```
Comment (spinach) [inline fastlane/.gitignore]: You're introducing a gitignore here, whereas we are adding stuff like `fastlane/test_output` for the .gitignore at the root level. Can we stick with one to have everything in one place?

File: fastlane/Appfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# The Appfile can be used to specify information that's used across all fastlane
+# tools, like your username or the app's bundle identifier.
+#
+# For more details, check out the documentation at:
+# https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Appfile.md
+
+ app_identifier "com.algolia.AlgoliaSearch" # the bundle identifier of your app
```
Comment (spinach) [inline fastlane/Appfile]: i quickly checked the Appfile.md, That's probably all we need to specify 👍

File: fastlane/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+<td width="33%" align="center">macOS or Linux with Ruby 2.0.0 or above</td>
+</tr>
+<tr>
+<td width="33%"><code>brew cask install fastlane</code></td>
+<td width="33%"><a href="https://download.fastlane.tools">Download the zip file</a>. Then double click on the <code>install</code> script (or run it in a terminal window).</td>
+<td width="33%"><code>sudo gem install fastlane -NV</code></td>
+</tr>
+</table>
+
+# Available Actions
```
Comment (spinach) [inline fastlane/README.md]: I think this ReadMe is cool to have we should keep :)

**Comment by spinach** [comment]
@robertmogos I'll let you close this one if you don't need anything from here

**Comment by robertmogos** [comment]
@spinach Yup, we'll discard it since is already merged

**Pull request closed without merging**